ios/Classes/PluginConfig.swift

```diff
@@ -1,30 +1,191 @@
+/* Copyright Airship and Contributors */
 import Foundation
 import AirshipKit
+import SwiftProtobuf
+
 
 class PluginConfig {
```

Comment:
It looks like this class isn't being used any more, unless I'm just missing where it's used. Do we still need it?

Reply:
True